spec/OBD/OBD.vspec

```diff
@@ -68,85 +75,99 @@
   description: PID 04 - Engine load in percent - 0 = no load, 100 = full load
 
 - CoolantTemperature:
-  datatype: float
+  datatype: int16
   type: sensor
   unit: celsius
+  min: -40
+  max: 215
   description: PID 05 - Coolant temperature
 
 - ShortTermFuelTrim1:
-  datatype: int8
+  datatype: float
   type: sensor
   unit: percent
   min: -100
-  max: 100
+  max: 99.2
   description: PID 06 - Short Term (immediate) Fuel Trim - Bank 1 - negative percent leaner, positive percent richer
 
 - LongTermFuelTrim1:
-  datatype: int8
+  datatype: float
   type: sensor
   unit: percent
   min: -100
-  max: 100
+  max: 99.2
   description: PID 07 - Long Term (learned) Fuel Trim - Bank 1 - negative percent leaner, positive percent richer
 
 - ShortTermFuelTrim2:
-  datatype: int8
+  datatype: float
   type: sensor
   unit: percent
   min: -100
-  max: 100
+  max: 99.2
   description: PID 08 - Short Term (immediate) Fuel Trim - Bank 2 - negative percent leaner, positive percent richer
 
 - LongTermFuelTrim2:
-  datatype: int8
+  datatype: float
   type: sensor
   unit: percent
   min: -100
-  max: 100
+  max: 99.2
   description: PID 09 - Long Term (learned) Fuel Trim - Bank 2 - negative percent leaner, positive percent richer
 
 - FuelPressure:
-  datatype: float
+  datatype: uint16
   type: sensor
   unit: kPa
   min: 0
+  max: 765
   description: PID 0A - Fuel pressure
 
 - MAP:
-  datatype: float
+  datatype: uint8
   type: sensor
   unit: kPa
   min: 0
+  max: 255
   description: PID 0B - Intake manifold pressure
 
 - EngineSpeed:
   datatype: float
   type: sensor
   unit: rpm
+  min: 0
+  max: 16383.75
   description: PID 0C - Engine speed measured as rotations per minute
 
 - Speed:
-  datatype: float
+  datatype: uint8
   type: sensor
   unit: km/h
+  min: 0
+  max: 255
   description: PID 0D - Vehicle speed
 
 - TimingAdvance:
   datatype: float
   type: sensor
   unit: degrees
+  min: -64
+  max: 63.5
   description: PID 0E - Time advance
 
 - IntakeTemp:
-  datatype: float
+  datatype: int16
```

Comment:
And the inconsistencies just continue. Here for #0F we currently use float, even if OBD only transmit integer values. But for #10 below we use int16 even if OBD sends the value with two decimals.